src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -25,6 +25,7 @@ class InputOption
     const VALUE_REQUIRED = 2;
     const VALUE_OPTIONAL = 4;
     const VALUE_IS_ARRAY = 8;
+    const VALUE_IS_COMMA_SEPARATED = 16;
```

Comment:
```php
$this->addInput('name', null, InputOption::VALUE_IS_ARRAY | InputOption::VALUE_IS_COMMA_SEPARATED);
```

Looks boring to me, as soon as the `VALUE_IS_COMMA_SEPARATED` option is only available with `VALUE_IS_ARRAY`.
I want to type:

```php
$this->addInput('name', null, VALUE_IS_COMMA_SEPARATED_ARRAY);
```

which is the same as typing `VALUE_IS_ARRAY | InputOption::VALUE_IS_COMMA_SEPARATED` (8|16 => 24)